kasa/smartdevice.py

```diff
@@ -27,6 +27,10 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+# Certain module queries will crash devices; this list skips those queries
+MODULE_EXCLUSIONS = {
+    "KL125(US)": ["antitheft", "cloud", "countdown"]
```

Comment:
Add a comment linking to the issue, just in case someone comes around wondering why this is done.

Btw, are you certain that `cloud` is also an issue? I'm seeing on my KL130 that it does not support `countdown`:
```
{'countdown': {'err_code': -2001, 'err_msg': 'Module not support'}
```
and one of the antitheft methods is also unsupported:
```
'smartlife.iot.common.anti_theft': {'get_next_action': {'err_code': -2000,
   'err_msg': 'Method '
   'not '
   'support'},
   'get_rules': {'enable': 0,
         'err_code': 0,
         'rule_list': []}},
```
while the cloud query has worked on all on my test devices.

Could it just be, that the number of modules (and/or methods) is the thing that breaks it?

Reply:
> Add a comment linking to the issue, just in case someone comes around wondering why this is done.

Done!

> Btw, are you certain that cloud is also an issue? I'm seeing on my KL130 that it does not support countdown:

I went back and did more testing and made a few additional interesting discoveries (that I'll document here in case someone else stumbles onto them).
- I had a power outage last week and so all of my bulbs were powered off about 72 hours ago.
- As a result, I discovered that querying the `cloud`, `countdown`, and `antitheft` modules did NOT cause most of my KL125 bulbs to consistently crash.  This tells me that the "crashing" of the bulbs is loosely tied to how long they've been powered on.
- I was able to get one of my bulbs to misbehave and confirmed your comment that the `countdown` and `antitheft` modules are indeed not supported, and the `cloud` module IS supported; however querying this module made the bulb crash with every other query.  
- So while the `cloud` module `is_supported`, it's interrogation seems to cause instability in the bulb.
- I was able to reduce the exclude list down to _just_ `cloud` and the bulb queries stabilized

I've reduced the exclusion list to `cloud` and that seems to be stable now for me.